src/guide/components/slots.md

```diff
@@ -296,6 +296,36 @@ function BaseLayout(slots) {
 }
 ```
 
+## Conditional Slots
+
+Sometimes you want to render something based on whether or not a slot is present. 
+
+You can use the [$slots](https://vuejs.org/api/component-instance.html#slots) component
+instance in combination with a [v-if](https://vuejs.org/guide/essentials/conditional.html#v-if) to achieve this.
```

Comment:
**suggestion**: I believe this requires a little rephrasing

```suggestion
You can use the [$slots](https://vuejs.org/api/component-instance.html#slots) property in combination with a [v-if](https://vuejs.org/guide/essentials/conditional.html#v-if) to achieve this.
```

Reply:
I've changed It to your recommendation!